src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import './Dropdown.scss';
+import cn from 'classnames';
+import { debounce } from 'lodash';
```

Comment:
Please chek the linter error and try to fix
https://github.com/mate-academy/react_autocomplete/actions/runs/5495363361/jobs/10014713836?pr=86

Reply:
fixed ✅ added lodash and fontawesome to dependenses